### ISSUE TITLE
filterEntities not unique with multiple cards

### ISSUE THREAD

**Issue by eggman0131** [alexarch21/history-explorer-card#61]
Hi, great card.

Not sure if I'm doing something wrong, or if there's a bug.  I've got two cards with unique names.  I have two different  filterEntities strings, both work fine in the card editor preview, but once I save the page both cards show the first list in the entities drop down.

code below

```
      - type: custom:history-explorer-card
        header: hide
        cardName: power
        recordedEntitiesOnly: true
        filterEntities: '*socket_power'
        defaultTimeRange: 1h
        combineSameUnits: true
        lineMode: stepped
        lineGraphHeight: 300
      - type: custom:history-explorer-card
        header: hide
        cardName: kwh
        recordedEntitiesOnly: true
        filterEntities: '*socket_daily*'
        defaultTimeRange: 5d
        combineSameUnits: true
        lineMode: stepped
        lineGraphHeight: 300

```

**Comment by alexarch21** [comment]
Confirmed and fixed in current dev build. Fix will be in the next release.

**Comment by eggman0131** [comment]
fab, thanks.  quick response much appreciatied

**Comment by alexarch21** [comment]
Fixed in V 1.0.26.
